roles/postgres/meta/argument_specs.yml

```diff
@@ -0,0 +1,28 @@
+---
+argument_specs:
+  setup_db:
+    short_description: Configure PostgreSQL database
+    options:
+      postgres_db_name:
+        type: str
+        required: true
+        description: |
+          Name of the database to be created
+      postgres_db_username:
+        type: str
+        required: true
+        description: |
+          Username of the database user
+      postgres_db_password:
+        type: str
+        required: true
+        description: |
+          Password of the database user
+      postgres_db_clients:
+        type: list
+        elements: str
+        required: true
+        description: |
+          List of clients that are allowed to connect to the database
```

Comment:
```suggestion
          List of clients addresses in CIDR notation that are allowed to connect to the database
```

?

Reply:
nope, it's actually expecting the inventory hostname list (e.g. the inventory group(s))=